printer_mods/README.md

```diff
@@ -118,6 +118,7 @@ like so:
 | tayto-chip | [V2.4 Skirt Switch Mod](./tayto-chip/skirt_switch_mod) | Add a ON/OFF Switch to side or front skirts| :x: | :x: | :heavy_check_mark: |:x:|
 | warriorofwire | [Slenderfan for Afterburner](./warriorofwire/slenderfan) | Lighter, quieter, smaller Afterburner | :x: | :x: | :heavy_check_mark: |:x:|
 | Weaslus | [Electronics Bay and Enclosure Seperator](./Weaslus/Electronics-Bay-Seperator) | Better enclose chamber and keep heat from electronics | :heavy_check_mark: | :x: | :x: |:x:|
+| TheOtherGuy110 | [Printable Back Panel Alternative](./TheOtherGuy1/Back_Panel_Alternative_Mod) | Works in tandem with Weaslus' Electronics Bay and Enclosure Seperator to replace stock back panel while maintaining easy access to steppers and RPi | :heavy_check_mark: | :x: | :x: |:x:|
```

Comment:
Please move this line up in the README so it's in alphanumeric order by name.

Please fix the link (your directory is `TheOtherGuy110`, but your link goes (and doesn't work) to `TheOtherGuy1`.  Your link should be

    (./TheOtherGuy110/Back_Panel_Alternative_Mod)